app/views/citizens/other_assets/_second_home_conditional_checkbox.html.erb

```diff
@@ -0,0 +1,24 @@
+
+<%
+  check_box_attribute = 'check_box_second_home'
+  conditional_div_id = 'conditional_second_home'
+  hint = I18n.translate("helpers.hint.#{check_box_attribute}", default: nil)
+  value1 = model.second_home_value
+  value2 = model.second_home_mortgage
+  value3 = model.second_home_percentage
```

Comment:
Can more meaningful names be used here rather than `value(n)`. `sh_value`, `sh_mortgage`, `sh_percentage` perhaps if the full names are really problematic.